src/guides/v2.3/frontend-dev-guide/validations/rule-list.md

```diff
@@ -0,0 +1,1664 @@
+...
+</form>
+```
+
+### validate-alpha
+
+It checks for letters (a-z or A-Z) only input.
+
+#### Example
+
+```html
+<form class="form" id="my-form" method="post" data-mage-init='{"validation": {}}'>
+...
+ <input name="field-4" id="field-3" title="Field 3" type="text" data-validate="{'validate-alpha':true}"/>
+...
+</form>
+```
+
+*  `jkjkjk` - true
+*  `dfdfdf1` - false
+
+### validate-alphanum
+
+It checks for letters (a-z or A-Z) or numbers (0-9) only input.
+
+#### Example
+
+```html
+<form class="form" id="my-form" method="post" data-mage-init='{"validation": {}}'>
+...
+ <input name="field-4" id="field-3" title="Field 3" type="text" data-validate="{'validate-alphanum':true}"/>
+...
+</form>
+```
+
+### validate-alphanum-with-spaces
+
+It checks for letters (a-z or A-Z) or numbers (0-9) or spaces only input.
+
+#### Example
+
+```html
+<form class="form" id="my-form" method="post" data-mage-init='{"validation": {}}'>
+...
+ <input name="field-4" id="field-3" title="Field 3" type="text" data-validate="{'validate-alphanum-with-spaces':true}"/>
+...
+</form>
+```
+
+### validate-cc-cvn
+
+It checks for a valid credit card identification number.
+
+#### Example
+
+```html
+<form class="form" id="my-form" method="post" data-mage-init='{"validation": {}}'>
+...
+  <input name="field-4" id="field-3" title="Field 3" type="text" data-validate="{'validate-cc-cvn':'#field-5'}"/>
+  <input name="field-5" id="field-5" type="text" value="AE"/>
+...
+</form>
+```
+
+### validate-cc-exp
+
+It checks for a valid credit card expiration month.
+
+#### Example
+
+```html
+<form class="form" id="my-form" method="post" data-mage-init='{"validation": {}}'>
+...
+  <input name="field-4" id="field-3" title="Field 3" type="text" data-validate="{'validate-cc-exp':'#field-5'}"/>
+  <input name="field-5" id="field-5" type="text" value="2022"/>
+...
+</form>
+```
+
+It interprets input as a month number with year set in a predefined field. Sanity check is limited.
+
+*  `40` - true
+*  `G` - false
+
+### validate-cc-number
+
+It checks for a valid credit card number based on mod 10.
+
+#### Example
+
+```html
+<form class="form" id="my-form" method="post" data-mage-init='{"validation": {}}'>
+...
+  <input name="field-4" id="field-3" title="Field 3" type="text" data-validate="{'validate-cc-number':true}"/>
+...
+</form>
+```
+
+### validate-cc-type
+
+It checks for a credit card number to match a predefined credit card type.
+
+#### Example
+
+```html
+<form class="form" id="my-form" method="post" data-mage-init='{"validation": {}}'>
+...
+  <input name="field-4" id="field-3" title="Field 3" type="text" data-validate="{'validate-cc-type':'#field-5'}"/>
+  <input name="field-5" id="field-5" type="text" value="AE"/>
+...
+</form>
+```
+
+Field `field-5` holds a credit card type, possible values are: `SO`, `SM`, `VI`, `MC`, `AE`, `DI`, `JCB`, `DN`, `UN`, `MI`, `MD`.
```

Comment:
As there are so many and some are not obvious - I'd recommend turning this list into a small table and defining what each of the 2 letter codes are.

Reply:
Fixed.